### DISCUSSION TITLE
How might the Cyber Resilience Act affect the development of Open-Source software?

### DISCUSSION THREAD

**Discussion by Lorenzolupoli** [community/community#55617] [Actions]
### Select Topic Area

General

### Body

I'm a second year student in a master's degree. currently, I'm writing a thesis on cybersecurity, my research question in the one outlined above, i want to open a discussion on the issue with authorities in the field. maybe, open a discussion forum is the best available option to gain some insights and different perspectives on the topic.

**Comment by stepcellwolf** [comment]
Hi,

Did you managed to get some input and opinions? Or at least graduated, as I can see it has been almost a year. It will be nice if you could share or we can discuss the CRA vs OOS.

Looking forward hearing back.
